docs/wiki/user/ace3-and-acex.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: wiki
-title: ACE3 and ACEX
+title: ACE3 And ACEX
```

Comment:
`And` should probably remain `and`

Reply:
Also felt wrong to me but didn't know if it was supposed to be like that